evm_integration.py

```diff
@@ -39,6 +38,8 @@ def main():
     adapter_factory_contract: {latest_env['adapterFactory']}
     legacy_tx: false
     poll_interval: 1
+  solana:
```

Comment:
maybe we should rename this to something like `development-solana` or `localnet`

Reply:
and can we rename this file to `chain_integration.py` or `test_chain_integration.py`? or move the solana stuff to a separate `svm_integration.py` file?